api/V1/recordingUtil.js

```diff
@@ -63,6 +63,7 @@ async function query(request, type) {
     request.query.limit,
     request.query.order
   );
+  builder.query.distinct = true;
```

Comment:
Does this have any effect on the query results themselves, or just the count?

Reply:
Looking at the query that is run it seems to just be the count, saw a few posts suggesting to do this online.

Reply:
Cool, sounds good